src/main/webapp/app/entities/programming-exercise/programming-exercise-instruction.component.ts

```diff
@@ -77,7 +77,8 @@ export class ProgrammingExerciseInstructionComponent implements OnChanges, OnDes
         private appRef: ApplicationRef,
         private injector: Injector,
     ) {
-        this.markdown = new Remarkable();
+        // Enabled for color picker of markdown editor that inserts spans into the markdown
+        this.markdown = new Remarkable({ html: true });
```

Comment:
We needed to enable html parsing in the programming exercises to make the color picker work.